src/main/resources/swagger.api/edge-dcb.yaml

```diff
@@ -1,123 +1,138 @@
 openapi: 3.0.0
 info:
   title: Edge DCB API
-  version: v1
+  version: 0.0.1
 servers:
   - url: /dcbService/
 paths:
-  /Transactions/{dcbTransactionId}/status:
-    parameters:
-      - in: path
-        name: dcbTransactionId
-        description: The ReShare DCB ID for the transaction being brokered.
-        required: true
-        schema:
-          type: string
-      - $ref: "#/components/parameters/x-okapi-token"
-      - $ref: "#/components/parameters/x-okapi-tenant"
+  /transactions/{dcbTransactionId}:
+    post:
+      summary: Create circulation request
```

Comment:
Please correct it to Create Transaction Request

Reply:
fixed